### ISSUE TITLE
GetAnotherServerAuthCode doesn't honor OAuth scope

### ISSUE THREAD

**Issue by FodderMK** [playgameservices/play-games-plugin-for-unity#2442]
I am configuring Google Play with the following:

```
PlayGamesClientConfiguration config = new PlayGamesClientConfiguration.Builder()
    .AddOauthScope("profile")
    .RequestServerAuthCode(false)
    .Build();
```
The server auth code I get after authenticating the user has the proper scope.  When I try get a new code with the following I get a server auth code that does NOT have the proper scope:
```
PlayGamesPlatform.Instance.GetAnotherServerAuthCode(true, code => {
    [...]
});
```
I suspect it is because `TokenFragment` uses `null` for the scope instead of what I previously specified:
```
public static PendingResult getAnotherAuthCode(Activity parentActivity,
                                               final boolean reauthIfNeeded,
                                               String webClientId) {
    return fetchToken(parentActivity,
                      /* silent= */!reauthIfNeeded,
                      /* requestAuthCode= */true,
                      /* requestEmail= */false,
                      /* requestIdToken= */false,
                      /* webClientId= */webClientId,
                      /* forceRefreshToken= */false,
                      /* additionalScopes= */null,
                      /* hidePopups= */true,
                      /* accountName= */null);
}
```

Ideally my previous scope would be used whenever I fetch another token but barring that at least having the option to pass in the scope again would also solve the issue.  If this is the expected behavior it would be good to document it.

Right now I have to log the user out and back in again in order to get another token with the proper scope.

**Comment by nindim** [comment]
I'm also seeing this behaviour.

It's causing my second login attempt to Playfab to fail with error: GoogleOAuthNoIdTokenIncludedInResponse

**Comment by olehkuznetsov** [comment]
Should be fixed in v0.9.61

**Comment by nindim** [comment]
Thank you for fixing this so quickly.
